release.sh

```diff
@@ -38,39 +38,12 @@ set -ex
 ROOTDIR="$(dirname "${BASH_SOURCE[0]}")"
 source "${ROOTDIR}/vendor/github.hpe.com/hpe/hpc-shastarelm-release/lib/release.sh"
 
-# Set to "unstable" for artifacts from main branch, "stable" for artifacts from
-# release branch.
-SAT_REPO_TYPE="unstable"
-SEMANTIC_VERSION_ONLY=${RELEASE_VERSION%%-*}
-MAJOR_MINOR_VERSION=${SEMANTIC_VERSION_ONLY%.*}
-
-DISABLE_GPG_CHECK=yes
-if [[ "${SAT_REPO_TYPE}" == "stable" ]]; then
-    DISABLE_GPG_CHECK=no
-fi
-
-# Substitute SAT variables
-source "${ROOTDIR}/component_versions.sh"
-for f in "${ROOTDIR}/docker/index.yaml" "${ROOTDIR}/cray-product-catalog/sat-component-versions.yaml" \
-    "${ROOTDIR}/install.sh" "${ROOTDIR}/rpm/sle-15sp2/index.yaml" "${ROOTDIR}/ansible/roles/sat-ncn/defaults/main.yml"
-do
-    sed -e "s/@SAT_REPO_TYPE@/${SAT_REPO_TYPE}/" \
-        -e "s%@ARTI_RPM_SUBDIR@%${ARTI_RPM_SUBDIR}%" \
-        -e "s/@SAT_VERSION@/${SAT_VERSION}/" \
-        -e "s/@SAT_PODMAN_VERSION@/${SAT_PODMAN_VERSION}/" \
-        -e "s/@RELEASE_VERSION@/${RELEASE_VERSION}/" \
-        -e "s/@CPCU_VERSION@/${CPCU_VERSION}/" \
-        -e "s/@CF_GITEA_IMPORT_VERSION@/${CF_GITEA_IMPORT_VERSION}/" \
-        -e "s/@DISABLE_GPG_CHECK@/${DISABLE_GPG_CHECK}/" \
-        -e "s/@SAT_INSTALL_UTILITY_VERSION@/${SAT_INSTALL_UTILITY_VERSION}/" \
-        -e "s/@CFS_CONFIG_UTIL_VERSION@/${CFS_CONFIG_UTIL_VERSION}/" \
-        -e "s/@DOCS_SAT_VERSION@/${DOCS_SAT_VERSION}/" \
-        $f.in > $f
-done
+"${ROOTDIR}/render_templates.sh"
 
 requires rsync sed realpath rpm2cpio
 
 BUILDDIR="${1:-"$(realpath -m "$ROOTDIR/dist/${RELEASE}")"}"
+SNYK_SCAN_DIR="${BUILDDIR}/scans"
```

Comment:
So the snyk scan results are actually packaged into our release distribution file? Is that intentional? Is there any potential for sensitive info to leak here?

I see below they're also being packaged into a separate tar.gz file. Was the intent to remove them from the `BUILDDIR` when done?

Reply:
I think in either case we would be publishing the information to artifactory, so I am not sure it's any worse to be packaging it directly in the release distribution.

I think this is the same as CSM/SMA are doing: https://github.com/Cray-HPE/csm/blob/main/release.sh#L306-L316

Similarly, I think it was intentional on the part of CSM to have the scans both in the release distribution as well as in a separate tar file.

Reply:
Actually, the SMA team has chosen to remove the scans from the release distribution: https://github.hpe.com/hpe/hpc-car-sma-product-stream/blob/release/sma-1.7/release.sh#L125-L126

I'd be happy to implement that change here if you think it's better, I'm personally not sure.

Reply:
I'm going to go ahead with that change, I don't there's any reason to have the information duplicated.

Reply:
I agree, thanks